### PR TITLE
Socket win32 x64 0.0.5 - CXX inject working

### PR DESCRIPTION
**Pull request by mribbons**

### CODE REVIEW

**Comment by mribbons** [comment]
I'll wait for at least one approval before I publish :)

**Comment by mribbons** [comment]
Fixes https://github.com/socketsupply/socket/issues/168

**Comment by mribbons** [comment]
This is live.

**Comment by mribbons** [comment]
This is live, go ahead from @heapwolf  via discord 👍

**Review by chicoxyzzy — COMMENTED** [review]
please fix linting so CI can pass

File: npm/packages/@socketsupply/socket-win32-x64/bin/install-pre-reqs.js
Comment (jwerle) [inline npm/packages/@socketsupply/socket-win32-x64/bin/install-pre-reqs.js:51]: can we handle the error case too?

```js
c.on('error', reject)
```

**Review by jwerle — CHANGES_REQUESTED** [review]
the linter should reveal other changes to make too

**Pull request merged**
